.puppeteerrc.cjs

```diff
@@ -0,0 +1,9 @@
+const {join} = require('path');
```

Comment:
This is needed because since Puppeteer > 19 chrome is installed in the home directory, so we have to move it to the base folder otherwise it's not packaged.